Fulfillment/Integration/02_ManagePackagingTypes.md

```diff
@@ -0,0 +1,206 @@
+
+    ![Packaging type updated](../../Assets/Screenshots/Fulfillment/Settings/PackagingTypes/UpdateSuccessful.png "[Packaging type updated]")
+
+    The *Edit packaging type* view is closed and the *List of packaging types* is displayed again.
+
+4. If necessary, click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button in the upper right corner to display the changes in the list.   
+  
+
+## Delete a packaging type
+
+Delete a packaging type that is no longer applicable. To avoid potential issues with existing shipments, it is highly recommended to deactivate a packaging type temporarily instead of deleting it permanently, see [Deactivate a packaging type](#deactivate-a-packaging-type).
+
+#### Prerequisites
+
+A packaging type has been created, see [Create a packaging type](#create-a-packaging-type).
+
+#### Procedure
+
+*Fulfillment > Settings > Tab PACKAGING TYPES*
+
+![Packaging types](../../Assets/Screenshots/Fulfillment/Settings/PackagingTypes/ListPackagingTypes.png "[Packaging types]")
+
+1. Click the checkbox of the packaging type to be deleted.  
+    The editing toolbar is displayed.
+
+    > [Caution] Deleting will permanently remove the selected data. The deletion cannot be undone and the deleted data cannot be restored. Problems may occur due to unresolved dependencies. 
+
+2. Click the [DELETE] button.  
+    The selected packaging type is deleted and removed from the list of packaging types.
+ 
+
+## Deactivate a packaging type
+
+Deactivate a packaging type temporarily, so that it can not be selected automatically by the system. 
+
+#### Prerequisites
+
+A packaging type has been created, see [Create a packaging type](#create-a-packaging-type).
+
+#### Procedure
+
+*Fulfillment > Settings > Tab PACKAGING TYPES*
+
+![Packaging types](../../Assets/Screenshots/Fulfillment/Settings/PackagingTypes/ListPackagingTypes.png "[Packaging types]")
+
+1. Click the packaging type to be deactivated. Alternatively, click the checkbox of the packaging type to be deactivated and the [EDIT] button in the editing toolbar.  
+    The *Edit packaging type* view is displayed.
+
+2. Disable the *Active* toggle.
+
+3. Click the [SAVE] button.   
+  The *Submitting data...* notice is displayed while saving followed by the *Update successful* pop-up window. 
+
+    ![Packaging type updated](../../Assets/Screenshots/Fulfillment/Settings/PackagingTypes/UpdateSuccessful.png "[Packaging type updated]")
+
+    The *Edit packaging type* view is closed and the *List of packaging types* is displayed again.
+
+4. If necessary, click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button in the upper right corner to display the changes in the list.   
+
+[comment]: <> (Stand 13.02.23: Inaktive Packaging types werden nicht angezeigt. Bug gemeldet)
+
+  
+## Activate a packaging type
+
+A packaging type that has been temporarily deactivated can be easily reactivated when it is needed again.
```

Comment:
```suggestion
A packaging type can only be used if it is active. If a packaging type has been deactivated , it can be reactivated again when it is needed.
```